.travis.yml

```diff
@@ -96,7 +96,8 @@ install:
   # Install cross-compilers for cross-compiling Linux build:
   - >
       if [[ "`uname`" == "Linux" && $DYNAMORIO_CROSS_ONLY == yes ]]; then
-      sudo apt-get -y install g++-arm-linux-gnueabihf g++-aarch64-linux-gnu; fi
+      sudo apt-get -y install g++-arm-linux-gnueabihf g++-aarch64-linux-gnu
+      linux-libc-dev-arm64-cross linux-libc-dev-armhf-cross gcc-arm-linux-androideabi; fi
```

Comment:
So the two libc ones are not needed, or they are needed for the root to point at?